WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -465,12 +467,14 @@ private extension OrderDetailsDataSource {
     private func configureShippingAddress(cell: CustomerInfoTableViewCell) {
         let shippingAddress = order.shippingAddress
 
-        cell.title = NSLocalizedString("Shipping details", comment: "Shipping title for customer info cell")
+        cell.title = NSLocalizedString("Shipping details",
+                                       comment: "Shipping title for customer info cell")
         cell.name = shippingAddress?.fullNameWithCompany
         cell.address = shippingAddress?.formattedPostalAddress ??
             NSLocalizedString(
                 "No address specified.",
-                comment: "Order details > customer info > shipping details. This is where the address would normally display."
+                comment: "Order details > customer info > shipping details. " +
```

Comment:
Just curiosity: the sum of two string is taken by our localization script?

Reply:
Yep! Concatenation using `+` works. More details in p77Llu-bLz-p2#comment-14876

Reply:
Yep! Concatenation using `+` works. More details in p77Llu-bLz-p2#comment-14876